packages/runed/src/lib/utilities/Lazy/Lazy.svelte.ts

```diff
@@ -0,0 +1,27 @@
+import { untrack } from "svelte";
+
+export class Lazy<T> {
+	#init: () => T;
+
+	constructor(init: () => T) {
+		this.#init = init;
+	}
+
+	#current: T | undefined = $state();
```

Comment:
```suggestion
	#current = $state<T>(null!);
```

Reply:
Why null tho? We can do this, no?

$state<T>()!

Reply:
TIL about the `$state()!`, I've been using `null!` because it will never have a type thus narrowing to the generic param but the training `!` is even better!

Reply:
You can also do `$state() as T`